index.html

```diff
@@ -1,183 +1,376 @@
+        <h3 class="best">
+          <img src="icons/Rectangle57.png" alt="circle" /> Skills
+        </h3>
+        <ul class="list">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="myresume">
+        <h3 class="h3-title">About me</h3>
+        <p class="about-paragraph">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button class="resume">Get My Resume</button>
+        <hr />
+      </div>
+      <div class="work-on">
+        <h3 class="best">
+          <img src="icons/Rectangle55.png" alt="rectangl" /> Language
+        </h3>
+        <ul class="list">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <h3 class="best">
+          <img src="icons/Rectangle56.png" alt="rectangle" /> Frameworks
+        </h3>
+        <ul class="list">
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>Rspec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <h3 class="best">
+          <img src="icons/Rectangle57.png" alt="circle" /> Skills
+        </h3>
+        <ul class="list">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+    <section class="contact-form">
+      <div class="the-contact">
+        <p class="contact-p">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+
+        <form action="https://formspree.io/f/" method="post">
+          <div class="input__sec">
+            <div class="name">
+              <label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="my-name"
+                  placeholder="Full Name"
+                  required
+                  minlength="2"
+                  maxlength="30"
+                />
+              </label>
             </div>
+            <p>
+              <label>
+                <input
+                  type="email"
+                  name="email-address"
+                  placeholder="Email address"
+                  required
+                />
+              </label>
+            </p>
+            <label>
+              <textarea
+                name="comment"
+                required
+                minlength="2"
+                maxlength="500"
+                placeholder="Hello Vistor! Glad to see you!"
+              ></textarea>
```

Comment:
Please try to follow the figma's design as much as possible 🤓 

#### Your design 
![Captura2](https://user-images.githubusercontent.com/15807343/149186761-a18a21a4-e352-4345-8943-38d205270985.PNG)

#### Figma's 
![Captura](https://user-images.githubusercontent.com/15807343/149186798-405c692a-b50c-4f02-91c3-4411538ff413.PNG)

What do you think if we use the same color on the text area to be as close as possible 🔥?

Reply:
Thanks bro for this and all the comments.